File: Service/TimesheetService/DataContracts/Result.cs
using System.Runtime.Serialization;

namespace TimesheetService.DataContracts {
    [DataContract]
    public class Result {
        [DataMember(Name="success")]
        public bool Success { get; set; }

        [DataMember(Name="data")]
        public object Data { get; set; }

        [DataMember(Name="errorMessage")]
        public string ErrorMessage { get; set; }
    }
}

File: Timesheet/src/com/aditi/timesheet/serializable/Result.java
package com.aditi.timesheet.serializable;

import com.aditi.timesheet.serializer.Serialize;

public class Result {
	@Serialize(name = "success")
	public boolean success;

	@Serialize(name = "data")
	public Object data;

	@Serialize(name = "errorMessage")
	public String errorMessage;
}